h/annotation_sync.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
`annotation_sync` seems like a bit of an odd name to me. Perhaps `h.indexer` or even `h.search`?